pkg/kubelet/probe.go

```diff
@@ -38,6 +41,27 @@ var (
 	tcprober   = tcprobe.New()
 )
 
+func (kl *Kubelet) probeLiveness(podFullName string, podUID types.UID, status api.PodStatus, container api.Container, dockerContainer *docker.APIContainers) (probe.Status, error) {
```

Comment:
given that these two are basically identical.  Can we refactor in terms of a helper function that takes a Probe as an argument, and then pass LivenessProbe or ReadinessProbe as appropriate?